numpy/__init__.pyi

```diff
@@ -128,6 +128,23 @@ from numpy.typing._callable import (
     _ComparisonOp,
 )
 
+# NOTE: Numpy's mypy plugin is used for removing the types unavailable
+# to the specific platform
+from numpy.typing._extended_precision import (
+    uint128 as uint128,
+    uint256 as uint256,
+    int128 as int128,
+    int256 as int256,
+    float80 as float80,
+    float96 as float96,
+    float128 as float128,
+    float256 as float256,
+    complex160 as complex160,
+    complex192 as complex192,
+    complex256 as complex256,
+    complex512 as complex512,
```

Comment:
Effectively the plugin removes all import statements that are invalid for the platform in question.

The full list of (potential) extended-precision types was taken from [`numpy/__init__.pxd`](https://github.com/numpy/numpy/blob/9f12028b9453c17b72b26355fd503e512af96a5d/numpy/__init__.pxd#L54-L79).